apps/sms/test/marionette/inbox_test.js

```diff
@@ -76,4 +76,69 @@ marionette('Inbox View tests', function() {
       });
     });
   });
+
+  suite('Split Inbox view tests', function() {
+    var numberOfConversations = 10;
+    var numberOfMessagesInConversation = 5;
```

Comment:
Ah didn't think that const works in tests now! And yeah with ```const``` all-caps will be better